homeassistant/components/voice_assistant/__init__.py

```diff
@@ -9,7 +9,8 @@
 
 from .const import DOMAIN
 from .error import PipelineNotFound
-from .pipeline import (
+from .pipeline import (  # pylint: disable=unused-import  # noqa: F401
```

Comment:
If this is being imported to be exported, add it to `__all__` to avoid having to add this comment

Reply:
Ah, didn't know that. Thanks.